components/VTheme/Profile.js

```diff
@@ -1,31 +1,56 @@
 import styled from 'styled-components'
 
 export const ProfileBanner = styled.header`
```

Comment:
I think we should rename profile banner to VBanner as we are using it consistently across all pages that have banners.

Reply:
They do differ though - but will refactor for upcoming profile changes as well